lib/core/cardano-wallet-core.cabal

```diff
@@ -118,6 +118,7 @@ test-suite unit
     , containers
     , cryptonite
     , deepseq
+    , extra >= 1.6.17
```

Comment:
To get `enumerate = [minBound..maxBound]`

Reply:
Agreed, it's a very small function!

But `[minBound .. maxBound]` seems to be surprisingly common.